jpos/src/main/java/org/jpos/core/Environment.java

```diff
@@ -69,6 +69,9 @@ private Environment() throws IOException {
     public String getName() {
         return name;
     }
+    public String getDir() {
```

Comment:
Given this is a public method, can we call it `getEnvDir()` instead of `getDir()` just to show we are talking about the 'environment' directory, and not Q2's CWD (which is actually the JVM's CWD), please?

Reply:
no problem, easy change

I tought it was implicit by virtue of having to call it on the `Environment` instance (such as the pre-existing `getName()` and could save a few characters